src/movement/thruster_sabertooth.cpp

```diff
@@ -84,12 +89,39 @@ void callBack(const robosub::thruster::ConstPtr& msg)
 
 }
 
+void setTimeOut (uint8_t ms_100)
+{
+  uint8_t serial_data[4];
+
+  //loop through 6 thrusters
+  for (int i = 0; i < 6; i++)
```

Comment:
Why six thrusters? I feel like this could be made more extensible so that if in the future the sabertooth thrusters are used again, but a different number of them. @irwineffect, what do you think?